fast/stages/02-networking-nva/data/subnets/prod/prod-dataplatform-ew1.yaml

```diff
@@ -0,0 +1,8 @@
+# skip boilerplate check
```

Comment:
do you still need this file in prod? (same for vpn)

Reply:
ah yes, users should clone if prod spins up